docker-compose-prod.yml

```
@@ -98,6 +98,7 @@ services:
       APP_GRAPHQL_ENDPOINT: ${FRONTEND_APP_GRAPHQL_ENDPOINT?error}
       APP_ENVIRONMENT: ${FRONTEND_APP_ENVIRONMENT:-prod}
       APP_GRAPHQL_CODEGEN_ENDPOINT: ${FRONTEND_APP_GRAPHQL_CODEGEN_ENDPOINT?error}
+      APP_GOOGLE_ANALYTICS_ID: ${APP_GOOGLE_ANALYTICS_ID?error}
```

Let's append FRONTEND_ 
```suggestion
      APP_GOOGLE_ANALYTICS_ID: ${FRONTEND_APP_GOOGLE_ANALYTICS_ID?error}
```